tests/smartswitch/test_show_platform_dpu.py

```diff
@@ -0,0 +1,183 @@
+"""
+Tests for the `platform cli ...` commands in DPU 
+"""
+
+import http.client
+import logging
+import pytest
+import time
+import redis
+from tests.common.utilities import wait_until
+from . import util
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.platform.device_utils_dpu import *
+from tests.common.helpers.platform_api import chassis, module
+from tests.platform_tests.api.conftest import platform_api_conn
+from tests.common.devices.sonic import *
+
+pytestmark = [
+    pytest.mark.topology('t1')
+]
+
+def test_midplane_ip(duthosts, enum_rand_one_per_hwsku_hostname, platform_api_conn):
+    """
+    @summary: Verify `Midplane ip address between NPU and DPU`
+    """
+    duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+    num_modules = int(chassis.get_num_modules(platform_api_conn))
+    ping_count = 0
```

Comment:
Both ping_count and dpu_count are unused here.

Reply:
removed it.